README.md

```diff
@@ -12,69 +13,11 @@ A Terraform Module to configure the Lacework Agentless Scanner.
 | Name | Version |
 |------|---------|
 | <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.15.0 |
-| <a name="requirement_aws"></a> [aws](#requirement\_aws) | ~> 4.0 |
-| <a name="requirement_lacework"></a> [lacework](#requirement\_lacework) | ~> 1.0 |
+| <a name="requirement_aws"></a> [aws](#requirement\_aws) | >= 4.0 |
+| <a name="requirement_lacework"></a> [lacework](#requirement\_lacework) | ~> 1.8 |
+| <a name="requirement_null"></a> [null](#requirement\_null) | >= 3.1.1 |
 | <a name="requirement_random"></a> [random](#requirement\_random) | >= 2.1 |
 
-## Providers
-
-| Name | Version |
-|------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | ~> 4.0 |
-| <a name="provider_lacework"></a> [lacework](#provider\_lacework) | ~> 1.0 |
-| <a name="provider_null"></a> [null](#provider\_null) | n/a |
-| <a name="provider_random"></a> [random](#provider\_random) | >= 2.1 |
-
-## Resources
```

Comment:
This looks good to me, just wondering why Resources isn't included in the sections?

Reply:
Hey @bebold-jhr - so sorry for the late response to this. Spoke with the team, and if we could put it back in (just in case) then I can go ahead and approve this and get it merged.

Reply:
Hey @wl-smith,

no problem. I've added the resources in the configuration and re-created the README.md.